project/src/main/java/com/google/sps/data/VolunteeringOpportunity.java

```diff
@@ -5,7 +5,7 @@
 
```

Comment:
Would you mind removing this change as well?

Reply:
ditto, linter passes